src/plugin/Makefile.in

```diff
@@ -353,7 +354,7 @@ HAS_PYTHON = @HAS_PYTHON@
 HAS_READLINE = @HAS_READLINE@
 HAS_SCREEN = @HAS_SCREEN@
 HAS_SCRIPT = @HAS_SCRIPT@
-HAS_SSH = @HAS_SSH@
+HAS_SSH_LOCALHOST = @HAS_SSH_LOCALHOST@
```

Comment:
This has been changed automatically by the latest configure.ac file. @gc00 had updated configure.ac but not this file in earlier pull request, because this variable is not used in source plugin, it's an artifact of automake.